lib/flatfile.ts

```diff
@@ -55,11 +55,13 @@ type WorkbookObject = {
 const BASE_PATH = "https://api.x.flatfile.com/v1";
 
 export const getRecordsByName = async ({
+  flowName,
   userId,
   workbookName,
   sheetName,
   spaceType,
 }: {
+  flowName: string;
```

Comment:
Let's make an enum type for this to have types help us here

```
enum FlowTypes = {
 Project = "project"
 }
 
```

etc

Reply:
edited